<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wupper</class>
 <widget class="QMainWindow" name="Wupper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wupper</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="3" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Average speed:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Board ID</string>
            </property>
           </widget>
          </item>
          <item row="25" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Console:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="getStatus">
            <property name="text">
             <string>get Wupper status</string>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>PLL LOCK</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Card ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_8"/>
          </item>
          <item row="4" column="5">
           <widget class="QLabel" name="label_blockcycles">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>FW version</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="label_average">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="20" column="1">
           <widget class="QPushButton" name="initWupper">
            <property name="text">
             <string>Reset Wupper</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QSpinBox" name="spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2048000</number>
            </property>
            <property name="value">
             <number>204800</number>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="3">
           <widget class="QPushButton" name="measureThroughput">
            <property name="text">
             <string>Start measurement</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>2</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="4">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>kB</string>
            </property>
           </widget>
          </item>
          <item row="7" column="5">
           <widget class="QPushButton" name="setBlockSize">
            <property name="text">
             <string>Set blocksize</string>
            </property>
           </widget>
          </item>
          <item row="25" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Core temperature(Â°C):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QLabel" name="label_temp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="20" column="3">
           <widget class="QGroupBox" name="groupBox_DMA_Read">
            <property name="title">
             <string>DMA Read</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Throughput measurement:</string>
            </property>
           </widget>
          </item>
          <item row="22" column="3">
           <widget class="QPushButton" name="startFullChain">
            <property name="text">
             <string>Start DMA read test</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Bytes transfered:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QGroupBox" name="groupBox_DMA_Write">
            <property name="title">
             <string>DMA Write</string>
            </property>
           </widget>
          </item>
          <item row="21" column="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Errors rate cycle:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QTextBrowser" name="textConsole">
            <property name="minimumSize">
             <size>
              <width>570</width>
              <height>270</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCustomPlot" name="customplot" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>290</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>290</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="customplot2" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>283</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>283</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_5">
   <property name="windowTitle">
    <string>toolBar_5</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
